init_autoloader.php

```diff
@@ -24,3 +24,7 @@
 if (!class_exists('Zend\Loader\AutoloaderFactory')) {
     throw new RuntimeException('Unable to load dependencies. Run `php composer.phar install`.');
 }
+
+//register doctrine extensions
+$classLoader = new \Doctrine\Common\ClassLoader('Numeric', __DIR__.'/extensions/doctrine');
+$classLoader->register();
```

Comment:
Is het per se nodig om dit globaal te doen? Kunnen het registreren van de extension, en de extension zelf niet in de foto module?